docs/introduction/faq.md

```diff
@@ -0,0 +1,216 @@
+
+Location verification is not required to participate in either (i) the global leaderboard or (ii) the regional leaderboard for the most competitive region (i.e., the region with the most storage).
+
+We consider the location of the storage and sealing hardware to be the location of the miner.
+
+Please note that miners who attempt to “spoof” their location will be completely ineligible for all rewards.
+
+### How will Filecoin identify the mining location of miners?
+
+When we verify miners’ locations, we’ll reach out by email and coordinate a time that the miner will be standing by to provide verification. We’ll ask for some specific evidence and require submission within a short time period.
+
+Location verification is not required to participate in either (i) the global leaderboard or (ii) the regional leaderboard for the most competitive region (i.e., the region with the most storage).
+
+We consider the location of the storage and sealing hardware to be the location of the miner.
+
+Please note that miners who attempt to “spoof” their location will be completely ineligible for all rewards.
+
+### Would the repair miner have a "budget" that it can spend on delegating new storage miners when previous ones fail? Or are tokens "refunded" when the original storage miner fails, and the repair miner just reallocates it as the budget for a new storage miner contract?
+
+Yes, the repair miner would charge a premium on the underlying storage, assuming some failure rate, and would be responsible for the extra costs if they chose unreliable storage miners to ultimately store the data.
+
+We descoped the repair miner work for mainnet launch as the network is complicated enough without it, and it's relatively straightforward as an upgrade later.
+
+### Does Filecoin reward network or main network line machine need a fixed IP?
+
+For mainnet, you will need a public IP address, but it doesn’t need to be fixed (just accessible).
+
+### What if we lost a sector accidentally, is there any way to fix that?
+
+If you lost the data itself, then no, there’s no way to recover that, and you will be slashed for it. If the data itself is recoverable, though (say you just missed a Windowed PoSt), then the Recovery process will let you regain the sector.
+
+### Has Filecoin confirmed the use of SDR algorithm? Is there any evidence of malicious construction?
+
+SDR is confirmed for mainnet launch, and we have no evidence of malicious construction. The algorithm is also going through both internal and external security audits.
+
+If you have any information about any potential security problem or malicious construction, reach out to our team at security@filecoin.org.
+
+### How likely is the NSE algorithm to come on line later? Is AMD's configuration the most consistent with the current SDR algorithm?
```

Comment:
How likely is it that the Filecoin protocol will switch to the NSE Proof-of-Replication construction later?